packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -217,6 +217,13 @@ class RabbitResourceTrackingStartedMessage(RabbitResourceTrackingBaseMessage):
     node_id: NodeID
     node_name: str
 
+    parent_project_id: ProjectID
+    root_parent_project_id: ProjectID
+    root_parent_project_name: str
+
+    parent_node_id: NodeID
+    root_parent_node_id: NodeID
```

Comment:
What is the difference between these?

Reply:
Project ID | Parent Project ID | Root Parent Project ID |
--- | --- | --- | 
A | NULL | NULL | 
B | A | A | 
C | B | A | 

As we support via API server to initialize projects inside of projects, the root parent project ID is the project ID from which the user initialized the computation. The same holds for the node inside of the project.

Reply:
Sorry so `parent` and `root_parent` are the same thing? These names are very confusing to me.